1-exercises/A-this/exercise-3.js

```diff
@@ -8,8 +8,12 @@
 
 let bankAccount = {
   currentAmount: 100,
-  deposit: function (amount) {},
-  withdraw: function (amount) {},
+  deposit: function (amount) {
+    return (this.currentAmount += amount);
+  },
+  withdraw: function (amount) {
+    return (this.currentAmount -= amount);
+  },
```

Comment:
Correct. What could be done to make your function more robust? Say negative input or withdraw more then the currenAmount?